build.zig

```diff
@@ -1,39 +1,59 @@
 const std = @import("std");
 const Build = std.Build;
 const GenerateDef = @import("build/GenerateDef.zig");
+const introspect = @import("build-introspect.zig");
 
 const aro_version = std.SemanticVersion{
     .major = 0,
     .minor = 0,
     .patch = 0,
 };
 
-fn addFuzzStep(b: *Build, target: std.zig.CrossTarget) !void {
+fn addFuzzStep(b: *Build, target: std.zig.CrossTarget, afl_clang_lto_path: []const u8, aro_module: *std.Build.Module) !void {
+    const fuzz_step = b.step("fuzz", "Build executable for fuzz testing.");
+    var fuzz_target = target;
+    fuzz_target.ofmt = .c;
+
+    const zig_lib_dir = try introspect.findZigLibDir(b.allocator, b.zig_exe);
```

Comment:
I think you should be able to use `b.zig_lib_dir` here.

Reply:
Looks like it's null by default - the other option would be to require the user to pass `-Dzig-lib-dir=<whatever>` on the command line

Reply:
Didn't notice that, the path is also available via `zig env | jq .lib_dir` (but implemented in build.zig of course).

Reply:
Updated it to lazily read from `zig env` if a fuzz build is requested.